<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddFile</class>
 <widget class="QDialog" name="AddFile">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>608</width>
    <height>277</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Conversion</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Input File and Mode</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="4" column="0">
        <widget class="QComboBox" name="cboxMode">
         <item>
          <property name="text">
           <string>Easy Mode</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Custom Mode</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="cboxEasyProfile">
         <item>
          <property name="text">
           <string>MP4 (MPEG4 + AAC)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MP4 (H.264 + AAC)</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QLineEdit" name="txtOutputDir">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLineEdit" name="txtInputFile">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="btnSetOutputDirectory">
         <property name="text">
          <string>Set Ouput Directory</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btnSelectFile">
         <property name="text">
          <string>Select File</string>
         </property>
        </widget>
       </item>
       <item row="4" column="3">
        <widget class="QComboBox" name="cboxContainer"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Video Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Video Codec&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QComboBox" name="cboxVideoCodec"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Audio Options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QComboBox" name="cboxAudioCodec"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Audio Codec&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Start conversion immediately</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddFile</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddFile</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
